### PR TITLE
Fabo/add staging api to csp

### PR DESCRIPTION
**Pull request by faboweb**
Closes #ISSUE

**Description:**

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

File: netlify.toml [hunk 1]
```diff
@@ -9,7 +9,7 @@
     '''
     Content-Security-Policy = '''
       default-src 'self' https://*.lunie.io https://*.netlify.app https://www.google-analytics.com https://monitoring.lunie.io:9000 https://js.intercomcdn.com;
-      connect-src 'self' https://api-iam.intercom.io https://widget.intercom.io wss://production-api.lunie.io/graphql https://production-api.lunie.io https://monitoring.lunie.io:9000;
+      connect-src 'self' https://api-iam.intercom.io https://widget.intercom.io wss://production-api.lunie.io/graphql https://production-api.lunie.io wss://staging-api.lunie.io/graphql https://staging-api.lunie.io https://monitoring.lunie.io:9000;
       img-src 'self' data: https://*.lunie.io https://*.netlify.app https://www.google-analytics.com https://s3.amazonaws.com https://lunie.fra1.digitaloceanspaces.com;
       script-src 'self' 'unsafe-inline' 'unsafe-eval' https://*.lunie.io https://*.netlify.app https://www.google-analytics.com https://api-iam.intercom.io https://js.intercomcdn.com https://widget.intercom.io;
```
Comment (faboweb) [inline netlify.toml:14]: I guess in the deployed build we should also not allow netlify.app

Comment (Bitcoinera) [reply netlify.toml:14]: I don't understand very well yet the Netlify architecture, that is why I left it there

File: netlify.toml [hunk 2]
```diff
@@ -9,7 +9,7 @@
     '''
     Content-Security-Policy = '''
       default-src 'self' https://*.lunie.io https://*.netlify.app https://www.google-analytics.com https://monitoring.lunie.io:9000 https://js.intercomcdn.com;
-      connect-src 'self' https://api-iam.intercom.io https://widget.intercom.io wss://production-api.lunie.io/graphql https://production-api.lunie.io wss://staging-api.lunie.io/graphql https://staging-api.lunie.io https://monitoring.lunie.io:9000;
+      connect-src 'self' https://api-iam.intercom.io https://widget.intercom.io wss://nexus-websocket-a.intercom.io wss://production-api.lunie.io/graphql https://production-api.lunie.io wss://staging-api.lunie.io/graphql https://staging-api.lunie.io https://monitoring.lunie.io:9000;
```
Comment (Bitcoinera) [inline netlify.toml]: Michiel also added this in his PR 😅 Well, better have it duplicate just in case?

Comment (faboweb) [reply netlify.toml]: I merged and used Michiels

**Pull request merged**